src/app/app-routing.module.ts

```diff
@@ -4,12 +4,14 @@ import { Component, NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { DashboardComponent } from './dashboard/dashboard.component'
 import { ProfileComponent } from './profile/profile.component'
+import { HowToComponent } from './navigation/how-to/how-to.component';
 
 const routes: Routes = [
   {path: 'dashboard', component: DashboardComponent},
   {path: 'profile', component: ProfileComponent},
   {path: 'enterbets', component: EnterBetsComponent},
   {path: 'about', component: AboutComponent},
+  {path: 'how-to', component: HowToComponent},
```

Comment:
this is what is displayed in the URL there should be no additional markings. Make this "howto" and it will display in the browser as /howto..